webextensions/api/tabs.json

```diff
@@ -2917,7 +2917,7 @@
                     "version_added": false
                   },
                   "firefox": {
-                    "version_added": false
+                    "version_added": 63
```

Comment:
This and line 1492 need to be strings. That's why the build fails for this PR. It's useful to run "npm run lint" locally before pushing your changes, then you can catch problems like this earlier.